setup.py

```diff
@@ -0,0 +1,52 @@
+from setuptools import setup
```

Comment:
This file could use some cleaning up

Reply:
Agreed, will do so in a different PR.

Reply:
See https://github.com/ESMValGroup/ESMValTool_sample_data/pull/8